actionmailer/.byebug_history

```diff
@@ -0,0 +1,8 @@
+exit
```

Comment:
I think this file shouldn't be here

If you think it's usefull -> https://help.github.com/articles/ignoring-files/#create-a-global-gitignore